src/main/java/ca/corefacility/bioinformatics/irida/config/services/IridaPluginConfig.java

```diff
@@ -23,7 +23,8 @@
 
 	private static final Logger logger = LoggerFactory.getLogger(IridaPluginConfig.class);
 
-	private static final Path PIPELINE_PLUGIN_PATH = Paths.get("/etc/irida/plugins");
+	@Value("${pipeline.plugin.path}")
+	String pipelinePluginPath;
```

Comment:
Switch this to a `private` variable to match the norm in the other config classes please.